src/main/main.c

```diff
@@ -159,11 +159,7 @@ void init(void)
     latchActiveFeatures();
 
 #ifdef ALIENFLIGHTF3
```

Comment:
All AlienFlight targets could make also use of this instead setting this in the target/config.c.